dockerfile/synse.dockerfile

```diff
@@ -9,66 +9,57 @@
 #   * full: a synse server build with emulator
 #
 
-#
-# BASE
-#
-FROM python:3.6-alpine as base
-LABEL maintainer="Vapor IO"
-
-
 #
 # BUILDER
 #
-FROM python:3.6-alpine as builder
+FROM vaporio/python:3.6 as builder
+COPY requirements.txt .
 
-RUN mkdir /build
 WORKDIR /build
 
-COPY requirements.txt /requirements.txt
-
-RUN set -e -x \
-    && apk --update --no-cache --virtual .build-dep add \
-        build-base \
-    && pip install --upgrade pip \
-    && pip install --prefix=/build -r /requirements.txt --no-warn-script-location \
-    && rm -rf /root/.cache \
-    && apk del .build-dep
+RUN pip install --prefix=/build -r /requirements.txt --no-warn-script-location \
+ && rm -rf /root/.cache
 
 
 #
 # SLIM
 #
-FROM base as slim
+FROM vaporio/python:3.6-slim as slim
 COPY --from=builder /build /usr/local
 
-RUN set -e -x \
-    && apk --update --no-cache add \
-        bash libstdc++ ca-certificates tini
+RUN apt-get update && apt-get install --no-install-recommends -y wget \
+ && wget https://github.com/krallin/tini/releases/download/v0.18.0/tini_0.18.0-amd64.deb \
+ && dpkg -i tini_*.deb \
+ && rm tini_*.deb \
```

Comment:
It looks like tini was added to 18.10 and beyond. So we should be able to get it available through apt eventually